src/deno_dir.rs

```diff
@@ -28,6 +29,17 @@ pub struct CodeFetchOutput {
   pub maybe_source_map: Option<Vec<u8>>,
 }
 
+lazy_static! {
+  // Using str as key, since MediaType does not impl Eq and Hash
+  static ref EXTENSION_MAP: HashMap<&'static str, msg::MediaType> = {
```

Comment:
Rather than using a static HashMap for this, I think it would be simpler to have a function: fn extmap(ext: &str) -> MediaType